index.js

```diff
@@ -20,8 +23,49 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+   
+    Recipe.create({title:"Pizza", level: "Amateur Chef", cuisine: "Italian", ingredients: ["flour", 'cheese', 'tomato sauce', 'pure magic']})
+    .then((recipe) => {
+      console.log(recipe.title)
+    })
+    .catch()
+
+    Recipe.insertMany(data)
+    .then((allRecipes) =>{
+
+      allRecipes.forEach((recipe) => {
+        console.log(recipe.title)
+      })
+      let updatePromise = Recipe.updateOne({title:"Rigatoni alla Genovese"}, {duration:100})
+      updatePromise.then(()=>{
+        console.log("updated")
+      })
+      let deletePromise= Recipe.deleteOne({title:"Carrot Cake"})
+      deletePromise.then(() =>{
+        console.log("SUCCESSSSS")
+
+      })
+
+      Promise.all([updatePromise, deletePromise])
+        .then(() => {
+           mongoose.connection.close()
+             .then(() => {
+               console.log(`Connection closed`)
+             })
+        })
+
+    })
+    .catch((recipe) =>{
```

Comment:
a catch will not get you the recipe, it will get you an error that you may want to console.log